hw/ip_templates/otp_ctrl/dv/otp_ctrl_sim_cfg.hjson.tpl

```diff
@@ -74,7 +74,7 @@
   uvm_test: otp_ctrl_base_test
   uvm_test_seq: otp_ctrl_base_vseq
 
-  run_opts: ["+cdc_instrumentation_enabled=1"]
+  run_opts: ["+cdc_instrumentation_enabled=1 +UVM_VERDI_TRACE=HIER +print_topology=1"]
```

Comment:
It is better to use dvsim run or build modes to configure things like verdi trace or print topology. That is particularly important since these settings are VCS specific and dvsim modes can become per-tool.